changelog/1.get_merged_prs.py

```diff
@@ -19,6 +19,8 @@ def parse_isoformat(string):
 
 REPOSITORY = 'astropy/astropy'
 
+YESTERDAY = datetime(2016, 12, 13, 0, 0, 0)
```

Comment:
This should be something like ``datetime.datetime.now()-datetime.timedelta(1)`` - will just add that to the branch